modules/migration-installing-cam-operator-ocp-4.adoc

```diff
@@ -8,66 +8,44 @@ ifdef::sourcecluster-4_1-4_x[]
 = Installing the Cluster Application Migration Operator on an {product-title} 4.1 source cluster
 
 You can install the Cluster Application Migration Operator on an {product-title} 4.1 source cluster with OLM.
-
-The Cluster Application Migration Operator installs Velero and Restic.
 endif::[]
 ifdef::sourcecluster-4_2-4_x[]
 = Installing the Cluster Application Migration Operator on an {product-title} 4.2 source cluster
 
 You can install the Cluster Application Migration Operator on an {product-title} 4.2 source cluster with OLM.
-
-The Cluster Application Migration Operator installs Restic and Velero.
 endif::[]
 
 ifdef::targetcluster-3-4,targetcluster-4_2-4_x,targetcluster-4_1-4_x[]
 = Installing the Cluster Application Migration Operator on an {product-title} {product-version} target cluster
 
 You can install the Cluster Application Migration Operator on an {product-title} {product-version} target cluster with OLM.
 
-The Cluster Application Migration Operator installs the following on the target cluster:
-
-* Migration controller CR
-* CAM web console
-* Restic
-* Velero
+The Cluster Application Migration Operator installs the CAM tool on the target cluster by default. If you want to install the CAM tool on a different cluster, you must update the Migration controller configuration so that Cluster Application Migration Operator does not install the CAM tool on this cluster.
```

Comment:
Needs a "the" in front of the second instances of "Cluster Application Migration Operator"